clients/drcachesim/scheduler/scheduler_impl.cpp

```diff
@@ -2727,6 +2763,9 @@ scheduler_impl_tmpl_t<RecordType, ReaderType>::update_next_record(output_ordinal
             VPRINT(this, 2, "output %d base timestamp = %zu\n", output,
                    outputs_[output].base_timestamp);
         }
+        // FIXME: When USE_INPUT_ORDINALS is enabled, this returns the input-local
```

Comment:
In my head USE_INPUT_ORDINALS is never set for a dynamic schedule: but I actually don't see the docs or code disallowing that.  I think we should disallow it.  Please list that as the preferred solution; maybe that downgrades this to XXX as no known use case would get here.